Sources/NIOCore/Utilities.swift

```diff
@@ -258,3 +258,18 @@ extension System {
     public static let supportsUDPReceiveOffload: Bool = false
     #endif
 }
+
+extension System {
+    /// Returns the UDP maximum segment count if the platform supports and defines it.
+    public static func udpMaxSegments() -> Int? {
```

Comment:
This should really be a `static let` that's computed on first access

Reply:
I'm not sure that's true: the cost of this function is pretty close to the cost of the `dispatch_once`, so we might prefer a static `var`.

Reply:
I've changed this to a `static var`.